### DISCUSSION TITLE
github Education pack can't verify

### DISCUSSION THREAD

**Discussion by pdgamage** [community/community#141960] [GitHub Education]
### Hi everyone,

I am in the Sri Lankan campus but when I go to get the git hub student pack, it says that it will confirm that I am not in Sri Lanka, what should I do?

**Comment by davevad93** [comment]
Hi @pdgamage ,

If you are submitting the application from your campus and are still unable to continue, it's possible that GitHub may have incorrect information about your school's location.

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
